DataStructures/SortedLinkedList/SortedLinkedListNode.cs

```diff
@@ -0,0 +1,32 @@
+﻿using System;
+
+namespace DataStructures.SortedLinkedList
+{
+    /// <summary>
+    /// TODO.
+    /// </summary>
+    /// <typeparam name="T">TODO. 2.</typeparam>
+    public class SortedLinkedListNode<T> where T : IComparable<T>
+    {
+        /// <summary>
+        /// Initializes a new instance of the <see cref="SortedLinkedListNode{T}"/> class.
+        /// TODO.
+        /// </summary>
+        /// <param name="data">TODO. 2.</param>
```

Comment:
Codacy found an issue: [Complete the task associated to this 'TODO' comment.](https://app.codacy.com/manual/siriak/C-Sharp/pullRequest?prid=6332606)